src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/FetchingTradeQuote.tsx

```diff
@@ -0,0 +1,24 @@
+import { Spinner } from '@chakra-ui/react'
+import type { SwapperName } from '@shapeshiftoss/swapper'
+import { useMemo } from 'react'
+
+import { TradeQuoteCard } from './components/TradeQuoteCard'
+
+export const FetchingTradeQuote = ({ swapperName }: { swapperName: SwapperName }) => {
```

Comment:
Do we need this as a new compnoent? couldn't `<TradeQuoteCard />` handle the fetching render flow internally?

Reply:
We could, but it would add complexity to it that doesnt benefit us, since quotes arent per swaper we wouldnt be able to keep it mounted and save on mount cost

Reply:
Managed to do it cleanly in fb0114e4b47a2c74ed33596859015fa2fe842d55